src/specific/s_clock.c

```diff
@@ -6,11 +6,11 @@
 #include <SDL2/SDL_timer.h>
 
 static Uint64 m_Counter = 0;
-static double m_Frequency = 0.0;
+static Uint64 m_Frequency = 0.0;
```

Comment:
I think it should be `0` or `0ull` or similar.

Reply:
Whoops, forgot this one - updating, sec